autoload/go.vim

```diff
@@ -0,0 +1,17 @@
+"Check if has vimproc
```

Comment:
Can you move this to `autoload/go/vimproc.vim` please. I keep vim-go related autoload files in the `autoload/go` folder.